config/hubs/carbonplan.cluster.yaml

```diff
@@ -108,10 +118,6 @@ hubs:
             userScheduler:
               enabled: false
           proxy:
-            service:
```

Comment:
@yuvipanda, can you explain a little bit more about these removals (proxy service/https and traefik)and why you needed to do it? Thanks!!